opencga-analysis/src/main/java/org/opencb/opencga/analysis/rga/RgaManager.java

```diff
@@ -67,12 +68,21 @@ public class RgaManager implements AutoCloseable {
 
     private final IndividualRgaConverter individualRgaConverter;
     private final GeneRgaConverter geneConverter;
+
     private final VariantRgaConverter variantConverter;
 
     private final Logger logger;
 
     private static final int KNOCKOUT_INSERT_BATCH_SIZE = 25;
 
+    private static Map<String, OpenCGAResult<?>> cacheMap;
+    private final int CACHE_SIZE;
+    private static final int DEFAULT_CACHE_SIZE = 1000;
+
+    static {
+        cacheMap = new HashMap<>();
```

Comment:
There is a nice trick for having a "LRU cache" with the LinkedHashMap:

```java
new LinkedHashMap<String, String>(1000) {
    @Override
    protected boolean removeEldestEntry(Map.Entry<String, String> eldest) {
        return size() > CACHE_SIZE;
    }
};
```

As the code says, it will start removing objects from the cache as soon as it reaches the expected size

Reply:
This can be thread-safe by using a `synchronizedMap`
```java
Collections.synchronizedMap(new LinkedHashMap<String, String>(1000) {
    @Override
    protected boolean removeEldestEntry(Map.Entry<String, String> eldest) {
        return size() > CACHE_SIZE;
    }
});
```

Reply:
> There is a nice trick for having a "LRU cache" with the LinkedHashMap:
> 
> ```java
> new LinkedHashMap<String, String>(CACHE_SIZE) {
>     @Override
>     protected boolean removeEldestEntry(Map.Entry<String, String> eldest) {
>         return size() > CACHE_SIZE;
>     }
> };
> ```
> 
> As the code says, it will start removing objects from the cache as soon as it reaches the expected size

This is just a quick temporal solution until we implement a better solution such as using a redis cache. We only want to store the first X elements and it is very likely that we don't reach the cache size.